index.html

```diff
@@ -23,16 +23,25 @@
     <link rel="stylesheet" type="text/css"
     media="screen and (min-width: 48em)"
     href="desktopstyle.css" />
+    <script src="script.js" defer></script>
   </head>
-  <body>
+  <body id="Porfolio">
+    <section class="mobile-menu hidden">
+      <a href="#" onclick="hide()">
+        <img src="images/Icon - Cancel.svg" id="cancel" alt="cancel">
+      </a>
+      <a href="#Portofolio" class="m-menu marg">Portfolio</a><hr>
```

Comment:
- [ ] You have done a great job to make sure that other options when clicked on lead to the exact section and the mobile menu close. However, I suggest that the portfolio also functions in the same way. You may consider portfolio to redirect to the homepage/section ☺️.